### ISSUE TITLE
Add aria-label to search input

### ISSUE THREAD

**Issue by kai687** [kai687/sphinxawesome-theme#10]
the input doesn't have an aria-label. Also, we have two times the same ID for the search-pane thing and the search input field.

**Comment by kai687** [comment]
It doesn't require an aria-label. It's pretty clear from voice over, what is happening. Maybe I can do something about the placeholder text though
